CHANGELOG.md

```diff
@@ -28,6 +28,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
     - Removes creation and deletion of DynamoDB record from API endpoint POST /rules/
   - **CUMULUS-2816**
     - Removes addition of DynamoDB record from API endpoint POST /collections
+  - **CUMULUS-2794**
+    - Remove API Collections model and all related test code
```

Comment:
Nit: we should include a note about removing the api endpoint for the migration counts.

Reply:
As far as I know, the only migration count tool we have is under the lambdas directory and it got referenced thru the api-client/migrationCounts (as noted below). Not sure if we have the tool via an api endpoint, unless im mistaken? Please let me know. Thanks

Reply:
As far as I know, the only migration count tool we have is under the lambdas directory and it got referenced thru the api-client/migrationCounts (as noted in line below). Not sure if we have the tool via an api endpoint, unless im mistaken? Please let me know. Thanks

Reply:
Added note for removing the api/endpoints/migrationCounts